### PR TITLE
Changes formatting of inflection summary. 

### PR DESCRIPTION
**Pull request by smkvr98**
Item from https://github.com/digitalpalitools/web-ui/issues/7 -
> line 1 syntax: pañña "a adj" like dīgha
 line 1 syntax for irreg: addasa 1 "addasa aor" irreg
 delete line 2 entirely - it is currently duplicated 3 times in each entry
 table can follow immediately afterwards

### CODE REVIEW

File: pls_core/src/inflections/mod.rs [hunk 1]
```diff
@@ -134,9 +134,17 @@ fn get_pali1_metadata(pali1: &str, host: &dyn PlsInflectionsHost) -> Result<Pali
...
         pm.inflection_class = inflection_class_from_str(inflection_class);
         pm.like = if !like.is_empty() {
-            format!("like {}", host.transliterate(like)?)
+            format!(
+                "{} like {}",
+                if inflection_class == "verb" {
+                    "conjugation"
+                } else {
+                    "declension"
+                },
```
Comment (parthopdas) [inline pls_core/src/inflections/mod.rs]: which part of the fix is this?

File: pls_core/src/inflections/snapshots/pls_core__inflections__tests__inflection_tests__conjugation_1_en.snap [hunk 2]
```diff
@@ -6,11 +6,8 @@ expression: html
 <div class="pls-inflection-root">
   <header class="pls-inflection-header">
     <summary>
-      <div class="pls-inflection-summary-word-info">^ābādheti$ &ndash; "eti pr" (like ^bhāveti$)</div>
-      <div class="pls-inflection-summary-definition">
-        <span class="pls-inflection-summary-definition-pos">(pr)</span> <span class="pls-inflection-summary-definition-meaning">oppresses, harasses</span>
-      </div>
```
Comment (parthopdas) [inline pls_core/src/inflections/snapshots/pls_core__inflections__tests__inflection_tests__conjugation_1_en.snap]: this needs to be pulled out of this commit - we're going to fix this differently.

ie host is going to pass an option whether it wants it or not. that way web-ui will have it and dictionary will not
see https://github.com/digitalpalitools/web-ui/issues/27

File: pls_core/src/inflections/templates/output.html [hunk 3]
```diff
@@ -2,13 +2,10 @@
...
       {% if pattern -%}
-      <div class="pls-inflection-summary-word-info">{{ pali1 }} &ndash; "{{ pattern }}" ({{ like }})</div>
+      <div class="pls-inflection-summary-word-info">{{ pali1 }} "{{ pattern }}" {{ like }}</div>
       {% else -%}
-      <div class="pls-inflection-summary-word-info">{{ pali1 }} ({{ like }})</div>
+      <div class="pls-inflection-summary-word-info">{{ pali1 }} {{ like }}</div>
       {% endif -%}
-      <div class="pls-inflection-summary-definition">
-        <span class="pls-inflection-summary-definition-pos">({{ pos }})</span> <span class="pls-inflection-summary-definition-meaning">{{ meaning }}</span>
-      </div>
```
Comment (parthopdas) [inline pls_core/src/inflections/templates/output.html]: needs to be pulled out of this PR as discussed above.

**Review by parthopdas — COMMENTED** [review]
please retain awareness of what you are fixing & what its priority is.

this commit itself should not be going in considering these are P2 issues. since you have done with work, please push in once the 3 issues are addressed.

File: pls_core/src/inflections/mod.rs (hunk 1 above)
Comment (smkvr98) [reply pls_core/src/inflections/mod.rs:143]: my bad. I should've looked at this revision - 
image

**Review by parthopdas — APPROVED** [review]

**Pull request merged**